dev-haskell/beam-postgres/beam-postgres-0.5.0.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
```

Comment:
Hm, these tests are working on my machine.

Reply:
are you running the test manually in the working directory, or using `ebuild .... test`?

Reply:
You're right, the tests are failing, either due to a sandbox issue or because they depend on postgreqsl-13 specifically.

```
Exception: SqlError {sqlState = "58P01", sqlExecStatus = FatalError, sqlErrorMsg = "could not open extension control file \"/usr/share/postgresql-13/extension/uuid-ossp.control\": No such file or directory", sqlErrorDetail = "", sqlErrorHint = ""}
```

Either way, the tests seem fragile and I will disable them.

Reply:
> You're right, the tests are failing

Is it the same for `gargoyle-postgresql`?